elements/rh-card/demo/inline-promo.html

```diff
@@ -0,0 +1,85 @@
+<rh-card variant="inline-promo">
+  <svg slot="inline-promo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 190">
```

Comment:
i'm not sure we should call this slot `inline-promo`. we are likely to introduce other variants (see #1188 ) so i think we'd be better off calling this slot `image`. WDYT @marionnegp ?

Reply:
Renamed the `inline-promo` slot to `image` in 75da1c7.

> Would a user have to add content to a slot?

No. If a slot doesn't have content, it gets: `display: none;`. Technically, they _should_ add content to the `body` slot.

> Is there a reason the image slot is an svg? Inline promos might not always use an svg image.

The SVG is just used for demo purposes. I have styled it to work just the same with an `<svg>` or `<img>`. You can [test it](https://github.com/RedHat-UX/red-hat-design-system/assets/575865/0e2b0994-45d0-4cd5-8524-c2e4c2bd8b6e) by adding an `<img>` in place of the SVG + a `slot="image"` attribute.